images/s6_assets/init/postgres-init

```diff
@@ -13,7 +13,7 @@ export C035="\e[35m"
 export C036="\e[36m"
 export C037="\e[37m"
 
-PGVERSION=$(postgres --version | grep -E -m 1 -o "1[0-9]")
```

Comment:
the `-m 1` originally came from here: https://github.com/pulp/pulp-oci-images/pull/522/files

If it doesn't work, then you can probably drop it completely